src/components/Card.js

```diff
@@ -1,41 +1,99 @@
+import Api from "../components/Api.js";
+import DeleteCardForm from "./DeleteCardForm.js";
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+    "Content-Type": "application/json",
+  },
+});
```

Comment:
Please, delete any import from class files

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.